packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -162,3 +154,43 @@ export const LocalCameraStream = AddEvents<typeof _LocalCameraStream, LocalCamer
 );
 
 export type LocalCameraStream = _LocalCameraStream & WithEventsDummyType<LocalCameraStreamEvents>;
+
+export const createMicrophoneStream = (constraints?: AudioDeviceConstraints) =>
+  wcmeCreateMicrophoneStream(LocalMicrophoneStream, constraints);
+
+export const createCameraStream = (constraints?: VideoDeviceConstraints) =>
+  wcmeCreateCameraStream(LocalCameraStream, constraints);
+
+export const createCameraAndMicrophoneStreams = (constraints?: {
+  video?: VideoDeviceConstraints;
+  audio?: AudioDeviceConstraints;
+}) => wcmeCreateCameraAndMicrophoneStreams(LocalCameraStream, LocalMicrophoneStream, constraints);
+
+export const createDisplayStream = (videoContentHint?: VideoContentHint) =>
+  wcmeCreateDisplayStream(LocalDisplayStream, videoContentHint);
+
+export const createDisplayStreamWithAudio = (videoContentHint?: VideoContentHint) =>
+  wcmeCreateDisplayStreamWithAudio(LocalDisplayStream, LocalSystemAudioStream, videoContentHint);
+
+export const createDisplayMedia = (
+  options: {
```

Comment:
shouldn't the type for `options` also be defined and exported by webrtc-core?

Reply:
The type used in the `createDisplayMedia` API in `webrtc-core` is slightly different from this type in that it includes the class constructors under `options.video` and `options.audio`. So this specific type is not shared between `webrtc-core` and `media-helpers`.